<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="verticalTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Configuration</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="6" column="0" colspan="4">
          <widget class="QNEGraphicsView" name="gvNodeEditor">
           <property name="renderHints">
            <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="cbNodeType">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAlphabetically</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Scroll to zoom, Right-Click or &lt;Del&gt; to remove, Ctrl+Click to select multiple nodes</string>
           </property>
           <property name="margin">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QDialogButtonBox" name="buttonBox_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="standardButtons">
               <set>QDialogButtonBox::Open|QDialogButtonBox::Save|QDialogButtonBox::SaveAll</set>
              </property>
              <property name="centerButtons">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="7" column="1" colspan="3">
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Zoom Out</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Zoom In</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Delete Selected</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalTabWidgetPage1">
        <attribute name="title">
         <string>Values</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="cbValues"/>
         </item>
         <item>
          <widget class="QTableView" name="twValues">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalTabWidgetPage2">
        <attribute name="title">
         <string>Implement/Sections</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Implements</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="5" column="3">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>149</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="btnImplMoveUp">
              <property name="text">
               <string>Move Up</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="btnImplRemove">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QCheckBox" name="cbImplConfirmation">
              <property name="text">
               <string>Confirmation</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="btnImplAdd">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="8">
             <widget class="QTreeView" name="tbImplements"/>
            </item>
            <item row="6" column="3">
             <widget class="QPushButton" name="btnImplApply">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QPushButton" name="btnImplMoveDown">
              <property name="text">
               <string>Move Down</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Sections</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="6" column="3">
             <widget class="QPushButton" name="btnSectApply">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>149</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" rowspan="8">
             <widget class="QTableWidget" name="tbSections"/>
            </item>
            <item row="3" column="3">
             <widget class="QPushButton" name="btnSectMoveUp">
              <property name="text">
               <string>Move Up</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QCheckBox" name="btnSectConfirmation">
              <property name="text">
               <string>Confirmation</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="btnSectRemove">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="btnSectAdd">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QPushButton" name="btnSectMoveDown">
              <property name="text">
               <string>Move Down</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Experimental Stuff</string>
        </attribute>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Camera Mode</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="rb_fixedCamera">
              <property name="text">
               <string>Fixed</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_firstPersonCamera">
              <property name="text">
               <string>First Person</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_orbitCamera">
              <property name="text">
               <string>Orbit</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Coordinate System</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../res/ressources.qrc">:/graphics/Coordinate-system_object-orientation.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QNEGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>qnegraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
